packages/plugin-browser-session/session.js

```diff
@@ -18,11 +18,6 @@ const sessionDelegate = {
       return sessionClient
     }
 
-    if (!sessionClient.config.endpoints.sessions) {
```

Comment:
The config schema now validates that `endpoints.sessions` must now be defined so this branch of code is obsolete.